client/src/components/application/SelectBox/index.js

```diff
@@ -47,10 +50,14 @@ export default class SelectBox extends React.Component {
 
 SelectBox.propTypes = {
   label: PropTypes.string.isRequired,
-  items: PropTypes.arrayOf(PropTypes.string).isRequired,
+  name: PropTypes.string.isRequired,
   className: PropTypes.string,
+  setval: PropTypes.func,
```

Comment:
I think this function should be required 
```suggestion
  setval: PropTypes.func.isRequired,
```